pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -152,20 +157,25 @@ def stylization(
         transformer = _transformer(
             style=style, impl_params=impl_params, instance_norm=instance_norm,
         )
-        if instance_norm or not impl_params:
-            # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/test.lua#L32
-            transformer = transformer.eval()
+    if instance_norm or not impl_params:
+        # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/test.lua#L32
+        transformer = transformer.eval()
     transformer = transformer.to(device)
 
     postprocessor = _postprocessor()
     postprocessor = postprocessor.to(device)
 
     with torch.no_grad():
-        content_transform = _content_transform(
-            impl_params=impl_params, instance_norm=instance_norm
-        )
-        content_transform = content_transform.to(device)
-        input_image = content_transform(input_image)
+        if impl_params:
+            if hyper_parameters is None:
+                hyper_parameters = _hyper_parameters(
+                    impl_params=impl_params, instance_norm=instance_norm
+                )
```

Comment:
We should always do this near the top of the function even if we only use it here. This makes it consistent with the other functions for example `training`.